Trappist/src/Promact.Trappist.Web/Startup.cs

```diff
@@ -141,7 +140,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             {
                 if (env.IsProduction())
                     context.Database.EnsureDeleted();
-                context.Database.Migrate();
+                //context.Database.Migrate();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS125)